_data/providers.json

```diff
@@ -438,7 +438,12 @@
       "name": "WitherHosting",
       "url": "https://witherhosting.com/",
       "description": "Get Geyser as a plugin. Put the port provided by the host in your config.yml for the Java and bedrock sections, or create a port in the ports manager on the panel and use that for bedrock with the main one for Java, and you will be able to connect! Refer to the hosts [support article](https://support.witherhosting.com/en/article/how-to-install-and-use-geysermc-1xn5l6v/) for more"
-    }
+    },
+    {
+      "name": "Hosterfy",
+      "url": "https://www.hosterfy.com/",
+      "description_template": "Get Geyser as a plugin, as in the setup guide. Use the same port as your Java server for the Bedrock port in your config (either by setting it yourself, or enabling “clone-remote-port”) and connect with the same IP and port as you would on Java. You can also ask for a new port."
```

Comment:
If you want to set a custom description (as you're doing here), you need to set the key to "description" instead of "description_template". The latter only applies for e.g. setting the "default" description template (defined at the top of the list)

Reply:
Indeed, I made a mistake! 
Fixed.